vars/provisionNodesV1.groovy

```diff
@@ -257,7 +257,7 @@ EOF'''
     provision_script += '''\nrm -f /etc/profile.d/openmpi.sh
                              rm -f /tmp/daos_control.log
                              yum -y erase metabench mdtest simul IOR compat-openmpi16
```

Comment:
Why not use `dnf` here also?  Mixing `yum` and `dnf` means duplicate package libraries including duplicate downloads of repo data.

There might be other uses of YUM that we want to switch to DNF.

Reply:
>  I think we want look at removing the tests using provisionNodesV1 instead of trying to optimize it.

daos-stack/pipeline-lib#287 does that IIRC.